pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -153,6 +153,11 @@ var (
 		CheckNodeMemoryPressurePred, CheckNodePIDPressurePred, CheckNodeDiskPressurePred, EvenPodsSpreadPred, MatchInterPodAffinityPred}
 )
 
+// Ordering returns the ordering of predicates.
```

Comment:
Moving the function here seems unnecessary

Reply:
Thanks for your feedback @draveness 
The reasons for moving here are:
1. closer to `predicatesOrdering` variable
2. avoid interrupting `CachedPersistentVolumeInfo` and it's method